docs/manager/graphql-reference/v2-schema.graphql

```diff
@@ -170,12 +468,112 @@ type PageInfo {
 }
 
 type Query {
+  _entities(representations: [_Any!]!): [_Entity]!
+  _service: _Service!
   artifact(id: ID!): Artifact
   artifacts(filter: ArtifactFilter = null, orderBy: [ArtifactOrderBy!] = null, before: String = null, after: String = null, first: Int = null, last: Int = null): ArtifactConnection!
   artifactGroup(id: ID!): ArtifactGroup
   artifactGroups(filter: ArtifactFilter = null, orderBy: [ArtifactOrderBy!] = null, before: String = null, after: String = null, first: Int = null, last: Int = null): ArtifactGroupConnection!
+  deployments(
+    filter: DeploymentFilter = null
+    orderBy: DeploymentOrderBy = null
+
+    """Returns the first n items from the list."""
+    first: Int = null
+
+    """Returns the items in the list that come after the specified cursor."""
+    after: String = null
+
+    """Returns the items in the list that come before the specified cursor."""
+    before: String = null
+
+    """Returns the items in the list that come after the specified cursor."""
+    last: Int = null
+  ): ModelDeploymentConnection!
+  deployment(id: ID!): ModelDeployment
+  revisions(
+    filter: ModelRevisionFilter = null
+    order: ModelRevisionOrder = null
+
+    """Returns the first n items from the list."""
+    first: Int = null
+
+    """Returns the items in the list that come after the specified cursor."""
+    after: String = null
+
+    """Returns the items in the list that come before the specified cursor."""
+    before: String = null
+
+    """Returns the items in the list that come after the specified cursor."""
+    last: Int = null
+  ): ModelRevisionConnection!
+  revision(id: ID!): ModelRevision
+  replica(id: ID!): ModelReplica
+}
+
+type RawServiceConfig {
+  config: JSONString!
+  extraCliParameters: String
+}
+
+type ReplicaManagement {
+  desiredReplicaCount: Int!
+  replicas: ModelReplicaConnection!
+}
+
+enum ReplicaStatus {
+  HEALTHY
+  UNHEALTHY
+}
+
+type ReplicaStatusChangedPayload {
+  replica: ModelReplica!
+}
+
+type ResourceConfig {
+  resourceGroup: ScalingGroupNode!
+  resourceSlots: JSONString!
+  resourceOpts: JSONString
+}
+
+input ResourceConfigInput {
+  resourceGroup: ScalingGroupNodeInput!
+  resourceSlots: JSONString!
+  resourceOpts: JSONString = null
+}
+
+type RoutingNode implements Node {
+  """The Globally Unique ID of this object"""
+  id: ID!
+  routingId: UUID!
+  endpoint: String!
+  session: UUID!
+  status: String!
+  trafficRatio: Float!
+  createdAt: DateTime!
+  errorData: JSONString!
+  liveStat: JSONString!
+}
+
+type SGLangServiceConfig {
+  config: JSONString!
 }
 
+type Scale {
+  autoScalingRules: [EndpointAutoScalingRuleNode!]!
+}
+
+extend type ScalingGroupNode @key(fields: "id") {
+  id: ID! @external
+}
+
+input ScalingGroupNodeInput {
+  name: String!
+}
+
+"""Different service configurations for model runtime"""
+union ServiceConfig = vLLMServiceConfig | SGLangServiceConfig | NVIDIAServiceConfig | MOJOServiceConfig | RawServiceConfig
```

Comment:
If not yet defined, please remove everything except RawServiceConfig and define it later.